<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MnistNet</string>
  </property>
  <widget class="QPushButton" name="btn_a">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>select file</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="main.qrc">:/newPrefix/9-1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_prediction">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>57</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NONE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PlotWidget" name="widget_graph" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>57</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Prediction: </string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
